lib/model/modelTranslate.dart

```diff
@@ -122,3 +122,26 @@ class EventTranslate {
     return _myEvent.locations?[0].room ?? "unknown";
   }
 }
+
+class StudyModuleTranslate {
```

Comment:
I would prefer to handle the internationalization of the app using a consumer-provider pattern. I have already created an issue for this. See #33.

Reply:
For now I can change the Translate classes to return I18NString instead of string and define an instance variable until this feature is implemented. Once it is there I can change it to use the provider.
Would this be sufficient?

Reply:
I think let's just keep it like this for now, we can make the necessary changes as soon as the providers are set up.

Reply:
I can handle internationalisation in a separate issue once this and other UI/UX issues are closed and the templates are created.